server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -23,11 +23,13 @@ def validate(deployment)
             deployment.errors[:rhev_engine_admin_password] << _('RHEV deployments must specify an admin password for the RHEV Engine')
           end
 
-          if deployment.rhev_storage_type.empty? or !['NFS', 'Local', 'Gluster'].include?(deployment.rhev_storage_type)
-            deployment.errors[:rhev_storage_type] << _('RHEV deployments must specify a valid storage type (NFS, Local, Gluster)')
+          if deployment.rhev_storage_type.empty? or !['NFS', 'Local', 'glusterfs'].include?(deployment.rhev_storage_type)
+            deployment.errors[:rhev_storage_type] << _('RHEV deployments must specify a valid storage type (NFS, Local, glusterfs)')
           end
 
-          if deployment.rhev_storage_type == 'NFS'
+          Rails.logger.warn("XXX #{deployment.rhev_storage_type}")
+
+          if deployment.rhev_storage_type == 'NFS' && deployment.rhev_storage_type != 'glusterfs'
```

Comment:
Is the second part of the check needed?   the check for != 'glusterfs'?  Same with line 65.